src/pages/groupComparison/labelComponents/GroupLegendLabelComponent.tsx

```diff
@@ -17,9 +17,9 @@ export interface IGroupLegendLabelComponentProps {
     datum?: any;
 }
 
-export const GroupLegendLabelComponent: React.FunctionComponent<
```

Comment:
[Keep type parameters inline for type annotations in variable declarations](https://prettier.io/blog/2019/11/09/1.19.0.html#keep-type-parameters-inline-for-type-annotations-in-variable-declarations-6467-by-sosukesuzuki)